objects/src/assets/testing.rs

```diff
@@ -13,6 +15,14 @@ pub struct NonFungibleAssetDetailsBuilder<T> {
     rng: T,
 }
 
+/// Builder for an `FungibleAsset`, the builder can be configured and used multiplied times.
+#[derive(Debug, Clone)]
+#[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
+pub struct FungibleAssetBuilder {
```

Comment:
We used to have mock structs serializable via serde so that we could write them to files - but I'm not sure if that's needed any longer. If not, we can probably remove `serde` derives.

Reply:
Removed